docs/configuration/snippets/snippet-000.ts

```diff
@@ -0,0 +1,74 @@
+import { createMachine } from "xstate";
+
+export const machine = createMachine({
```

Comment:
There should be a string that describes the diagram layout. It's generated when you open the diagram via the XState VS Code extension.